cmd/replay.go

```diff
@@ -267,3 +270,90 @@ func runReplayRequest(l logger, projectName, namespace, jobName, startDate, endD
 	}
 	return replayResponse.Id, nil
 }
+
+func replayStatusSubCommand(l logger, conf config.Provider) *cli.Command {
+	var (
+		replayProject string
+		namespace     string
+	)
+
+	reCmd := &cli.Command{
+		Use:     "status",
+		Short:   "get status of a replay using its ID",
+		Example: "optimus replay status replay-id",
+		Long: `
+This operation takes one argument, replay ID[required]
+that generated when starting a replay.
+		`,
+		Args: func(cmd *cli.Command, args []string) error {
+			if len(args) < 2 {
```

Comment:
In the description, you said we need 1 argument, here we are looking for 2.

Reply:
updated to only 1